docker/pypi/wmagent/run.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/bash
 
 ### Basic initialization wrapper for WMAgent to serve as the main entry point for the WMAgent Docker container
+wmaUser=$(id -un)
+wmaGroup=$(id -gn)
+wmaUserID=$(id -u)
+wmaGroupID=$(id -g)
+export WMA_USER=$wmaUser
```

Comment:
didn't you have to create the `/home` folder for the user in the `run.sh` script for the wmagent container as well?